### PR TITLE
[14.0][MIG]product_packaging_dimension

### PR DESCRIPTION
**Pull request by watthanun**

### CODE REVIEW

File: product_packaging_dimension/models/product_packaging.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        compute="_compute_volume",
+        readonly=True,
+        store=False,
+        help="volume in cubic meters",
+    )
+
+    @api.depends("lngth", "width", "height")
+    def _compute_volume(self):
+        for pack in self:
+            pack.volume = (pack.lngth * pack.width * pack.height) / 1000.0 ** 3
```
Comment (rousseldenis) [inline product_packaging_dimension/models/product_packaging.py:28]: IMHO, it's time to add a little test for this

**Review by rousseldenis — APPROVED** [review]
Code review. A test would be helpful.

File: product_packaging_dimension/models/product_packaging.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    max_weight = fields.Float("Weight (kg)")
+    # lngth IS NOT A TYPO https://github.com/odoo/odoo/issues/41353
+    lngth = fields.Integer("Length (mm)", help="length in millimeters")
+    # Although it feels weird to use Integer in millimeters, we use Int to
+    # override the fields from delivery module instead of defining new ones
+    width = fields.Integer("Width (mm)", help="width in millimeters")
+    height = fields.Integer("Height (mm)", help="height in millimeters")
```
Comment (atchuthan) [inline product_packaging_dimension/models/product_packaging.py:15]: The fields are already available in product.packaging
https://github.com/odoo/odoo/blob/14.0/addons/delivery/models/product_packaging.py

**Review by atchuthan — CHANGES_REQUESTED** [review]

**Comment by hparfr** [comment]
I think it's better to not hardcode UoM.

A similar work has been done in this PR https://github.com/OCA/product-attribute/pull/775

**Review by claudiagn — APPROVED** [review]
LGTM! :+1: 

https://user-images.githubusercontent.com/61511679/112846095-f20eec80-90a5-11eb-98d5-36797495f7ff.mp4

**Comment by rousseldenis** [comment]
@watthanun @kittiu Could we switch to the more recent one : https://github.com/OCA/product-attribute/pull/850 ?

**Comment by kittiu** [comment]
> @watthanun @kittiu Could we switch to the more recent one : #850 ?

Can you help close this one. Watthanun is no longer here. Thanks @rousseldenis

**Pull request closed without merging**